tests/containers/nodejs-attach/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# node:10.24.1
```

Comment:
As I suggested in https://github.com/Granulate/gprofiler/pull/451/files#r975877845, we don't need a separate Docker image, please remove it and after https://github.com/Granulate/gprofiler/pull/477 is merged, build the `tests/containers/nodejs/` image with a build argument that prevents it from using the jitdump file